src/styles/main.css

```diff
@@ -33,32 +26,28 @@ header {
   position: relative;
 }
 
+.task-input img {
+  width: 30px;
+  height: 30px;
+  position: absolute;
+  top: 24%;
+  left: 10%;
+}
+
```

Comment:
You have a beautiful design for your todo list. However, launching the project using the dev script (`npm run dev`) does not seem to compile the CSS properly as it does not have styling. This makes it impossible to use the beautiful features you have. Kindly fix this.

I could see the styling on by running the `src/index.html` file in live server.

Project on live server:
![image](https://user-images.githubusercontent.com/20554583/221200538-c5a4f79c-1e7c-4f43-b250-978be09c9bf8.png)

Launching with script:
![image](https://user-images.githubusercontent.com/20554583/221200592-a5269bed-e0ad-45dc-8752-534b556059c1.png)